pkg/kubectl/cmd/run.go

```diff
@@ -85,6 +85,10 @@ func NewCmdRun(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *c
 	cmd.Flags().Bool("attach", false, "If true, wait for the Pod to start running, and then attach to the Pod as if 'kubectl attach ...' were called.  Default false, unless '-i/--interactive' is set, in which case the default is true.")
 	cmd.Flags().String("restart", "Always", "The restart policy for this Pod.  Legal values [Always, OnFailure, Never].  If set to 'Always' a replication controller is created for this pod, if set to OnFailure or Never, only the Pod is created and --replicas must be 1.  Default 'Always'")
 	cmd.Flags().Bool("command", false, "If true and extra arguments are present, use them as the 'command' field in the container, rather than the 'args' field which is the default.")
+	cmd.Flags().String("cpu-request", "", "The CPU request for the container expressed in millicores.")
```

Comment:
We might want these grouped like https://github.com/kubernetes/kubernetes/blob/d23d083439d4b936ceaa82d562fc9880906412c4/pkg/kubectl/cmd/util/printing.go#L30